src/Infrastructure/Persistence/Doctrine/Fixtures/TokenFixture.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Fixtures;
+
+use App\Domain\User\Enum\TokenTypeEnum;
+use App\Domain\User\Token;
+use App\Domain\User\User;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+
+final class TokenFixture extends Fixture implements DependentFixtureInterface
+{
+    public function load(ObjectManager $manager): void
+    {
+        $forgotPasswordToken = new Token(
+            '8edb17c6-9346-4e64-8a0d-bf1e4762802c',
+            'forgotPasswordToken',
```

Comment:
(Détail) Mettre ça en const sur la classe et y faire référence dans les tests ?

```suggestion
            self::FORGOT_PASSWORD_TOKEN_VALUE,
```